shell/platform/embedder/embedder.h

```diff
@@ -2505,6 +2566,25 @@ FLUTTER_EXPORT
 FlutterEngineResult FlutterEngineRunInitialized(
     FLUTTER_API_SYMBOL(FlutterEngine) engine);
 
+//------------------------------------------------------------------------------
+/// @brief      Add a view.
```

Comment:
Uber-pedantic nit: "Adds a view." for consistency.

Reply:
Fixed!